### PR TITLE
user card 

### PR DESCRIPTION
**Pull request by fadeomar**
Relates: #46
crate user card component that contains a container div inside of it the user image and div for user information that contains 

1. the name of the user 
2. the pref data about the user 
3. native language
4.  native language 

----

### how to use : 
1-put the this staticData 
```js
const userInfo = [
  {
    id: 5,
    username: 'Fatmasiam',
    email: 'fatma.o.siam@gmail.com',
    nativeLang: 'English',
    learningLang: 'Arabic',
    avatar: null,
    bio: 'Full stack web developer ',
  },
  {
    id: 6,
    username: 'Fade',
    email: 'fade@gmail.com',
    nativeLang: 'English',
    learningLang: 'Arabic',
    avatar: null,
    bio: 'Full stack web developer ',
  },
];
```

2-  in the app.js file 
  * import UserCard from './components/commonComponents/UserCard';
 *
 ```js
       {userInfo.map(info => (
          <UserCard key={info.id} userInfo={info} />
        ))}
```

### CODE REVIEW

File: client/src/components/commonComponents/UserCard/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
```
Comment (Fatmasiam) [inline client/src/components/commonComponents/UserCard/index.js]: import the css file

File: client/src/components/commonComponents/UserCard/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+const UserCard = ({
+  userName,
+  imageSrc,
+  userBio,
+  nativeLanguage,
+  learningLanguage,
+}) => (
+  <div className="userCard">
+    <img scr={imageSrc} alt="userImage" className="userCard-image" />
```
Comment (Fatmasiam) [inline client/src/components/commonComponents/UserCard/index.js]: using a suitable BEM naming

Comment (Fatmasiam) [inline client/src/components/commonComponents/UserCard/index.js]: from where the imageSrc comming?

File: client/src/components/commonComponents/UserCard/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  <div className="user-card">
+    <img
+      src={avatar || 'https://img.icons8.com/clouds/2x/user.png'}
+      alt="userImage"
+      className="user-card__image"
+    />
+    <div className="user-card__content">
+      <h2>{username}</h2>
+      <div className="user-card__content__languages">
+        <div className="user-card__content__languages-native">
```
Comment (Amoodaa) [inline client/src/components/commonComponents/UserCard/index.js]: NICE BEM BABY

**Pull request merged**